lib/elixir/lib/file.ex

```diff
@@ -776,9 +778,9 @@ defmodule File do
   be a path to a non-existent file. If either is a directory, `{:error, :eisdir}`
   will be returned.
 
-  The `callback` function is invoked if the `destination_file` already exists.
-  The function receives arguments for `source_file` and `destination_file`;
-  it should return `true` if the existing file should be overwritten, `false` if
+  An optional `on_conflict` callback is invoked if the `destination_file` already
+  exists. The function receives arguments for `source_file` and `destination_file`.
+  It should return `true` if the existing file should be overwritten, `false` if
```

Comment:
Perhaps the to-be-deprecated clauses could be be mentioned in the documentation at least until it is officially deprecated? Otherwise they get on a limbo, where they are allowed to be used but not documented, which can be confusing: "If I'm giving a function as argument instead of a list of options, why does it work?"